test_scala_library_outputs_mixed_java_scala_jars.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+echoerr() {
+  echo "$@" 1>&2;
+}
+
+assert() {
+  $@ || (echoerr "FAILED: $@"; exit 1)
+}
+
+exists() {
+  [ -e $1 ]
+}
+
+output_contains() {
+  grep '{"id":{"targetCompleted' $1 | grep $2 > /dev/null
+}
+
+test_scala_library_outputs_mixed_java_scala_jars() {
+  set -e
+
+  TARGET="MixJavaScalaLib"
+  BUILD_EVENT_FILE="build_event.json"
+
+  bazel build test:"$TARGET" --build_event_json_file=$BUILD_EVENT_FILE
```

Comment:
@ittaiz this was the only way I could find to get the outputs from the command line. If you know of a better way, I'd love to hear about it.